packages/webpack-config/src/webpack/base/module.js

```diff
@@ -13,13 +13,13 @@ import { compileShadowCSS } from "./tasks/compileShadowCSS";
 import { loadHandlebars } from "./tasks/loadHandlebars";
 import { cleanDest } from "./tasks/cleanDest";
 import { tsTypeChecking } from "./tasks/tsTypeChecking";
-// CompileHTML
+import { copyResources as copyResourcesTask } from "./tasks/copyResources";
```

Comment:
when introducing named imports, you can also cleanup the `getAppConfig().cleanDest` below (line 30)

Reply:
done